src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -219,4 +219,10 @@
   <data name="SARIF1020_SchemaReferenceMissing" xml:space="preserve">
     <value>{0}: The SARIF log file does not contain a $schema property. Add a $schema property that refers to the final version of the SARIF 2.1.0 schema. This enables IDEs to provide Intellisense for SARIF log files.</value>
   </data>
+  <data name="SARIF1021_Default" xml:space="preserve">
+    <value>{0}: The tool driver version is missing.</value>
+  </data>
+  <data name="SARIF1021_MissingToolVersion" xml:space="preserve">
+    <value>The 'version' property for tool driver should be present.</value>
```

Comment:
>T [](start = 11, length = 1)

And then, very similar to the result message:
"The property 'tool.driver.version' should be present. This property is important in workflows that must verify the version of the tool being used."